project.md

```
@@ -62,6 +64,9 @@ In many fitness group training classes, the coach starts with a "Question of the
 - ⏸ Admin portal separation (currently co-located with consumer app)
 - ⏸ Feature flags to gate beta features (More Like This, Style/Tone drawers)
 
+### Feedback-informed curation preview
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)